src/lsptoolshost/uriConverter.ts

```diff
@@ -24,6 +24,6 @@ export class UriConverter {
     }
 
     public static deserialize(value: string): vscode.Uri {
-        return vscode.Uri.parse(value);
+        return vscode.Uri.parse(value, true);
```

Comment:
what does true do here?

Reply:
it's the `strict` parameter. It causes it to throw if the Uri doesn't have a schema